### DISCUSSION TITLE
How do I exclude Pathman from messing up things?

### DISCUSSION THREAD

**Discussion by budimanjojo** [webinstall/webi-installers#348] [Q&A] [answered]
I just installed `zoxide` using `webi` and it include `pathman` with it (Why?). 
Here's why it matter:
1. I already have `$HOME/.local/bin` in my `$PATH` (I use fish shell)
2. I install zoxide with this command: `curl -sS https://webinstall.dev/zoxide | bash`
3. Now my $PATH have duplicated entry
```
> echo $PATH
/home/budiman/.local/bin /home/budiman/.local/bin /usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin /usr/games /usr/local/games /snap/bin
```

What I needed is a simple flag that I can pipe into the command to exclude adding path, is that possible? Thanks before.

**Comment by coolaj86** [comment, marked as answer]
See https://github.com/webinstall/webi-installers/issues/322, https://github.com/webinstall/webi-installers/issues/653

**Reply by budimanjojo** [reply]
okay, so we will see that soon? I suggest using the flag doesn't install pathman completely, just install the binary for what we want 😊
